cli/compose/loader/full-example.yml

```diff
@@ -2,6 +2,13 @@ version: "3.4"
 
 services:
   foo:
+
+    build:
+      context: ./dir
+      dockerfile: Dockerfile
+      args:
+        foo: bar
```

Comment:
Please add the other fields (target, network, cache_from, labels) to this as well.